### DISCUSSION TITLE
How to reference metadata from _quarto.yml in html?

### DISCUSSION THREAD

**Discussion by zoushucai** [quarto-dev/quarto-cli#6748] [Q&A] [answered]
### Description

How to Import data from ` _quarto.yml` files in HTML files or qmd file

```qmd
project:
  type: website
  output-dir: _site 
website:
  sidebar:
    style: 'floating'

```
I want to introduce website.sidebar.style in the HTML file

But doesn't it work if I write like this?

```yml
{{< meta website.sidebar.style >}}

```

or qmd file?

```qmd
---
title: 'title'
author: 'zsc'
date: '2023-09-22'
---

{{< meta website.sidebar.style >}}

```

**Comment by dragonstyle** [comment]
Currently the project level namespaces like `book` and `website` aren't available in the document metadata, so they are not available to the `meta` shortcode either.

Can you say a bit more about what you're trying to do - we can maybe find a way to better support the effort...

**Comment by zoushucai** [comment]
Thank you. I would like to write a template and read variable information from the `_quarto.yml` file to display different effects based on the value of the variable,

**Comment by zoushucai** [comment]
I don't really want to create a new `_variable.yml` file, but I found that the variable I want exists in the `_quaro.yml ` file

**Reply by dragonstyle** [reply, marked as answer]
For many variables, this should be possible, but it isn't currently possible for keys under `website` or `book`.

**Comment by zoushucai** [comment]
Okay, thank you, I hope to use it someday
